src/Api/Controllers/OrganizationUsersController.cs

```diff
@@ -297,17 +297,17 @@ public async Task PutResetPasswordEnrollment(string orgId, string userId, [FromB
                 throw new UnauthorizedAccessException();
             }
 
-            if (model.ResetPasswordKey != null && !await _userService.VerifySecretAsync(user, model.Secret))
-            {
-                await Task.Delay(2000);
-                throw new BadRequestException("MasterPasswordHash", "Invalid password.");
-            }
-            else
-            {
-                var callingUserId = user.Id;
+            // if (model.ResetPasswordKey != null && !await _userService.VerifySecretAsync(user, model.Secret))
+            // {
+            //     await Task.Delay(2000);
+            //     throw new BadRequestException("MasterPasswordHash", "Invalid password.");
+            // }
+            // else
+            // {
```

Comment:
Is there a reason this is commented out instead of deleted?
Also, it looks like linting failed on this file

Reply:
This is being merged in directly to a release candidate as turning off a feature. I didn't want to delete since the intent in that master will keep this code as-is. It's actually what I did for #2084, which is similarly delayed to give clients a chance to catch up to API changes.

This is why the linter is failing.